File: Source/ArrayRelatedOperations/frequency.js
function findFrequency(a) {
    let freq = {};
    for(let i = 0; i < a.length; i++) {
        const key = a[i].toString();
        if(freq[key] === undefined) {
            freq[key] = 1;
        } else {
            freq[key]++;
        }
    }

    return freq;
}

File: Source/ArrayRelatedOperations/frequency.c
#define MAX 100000  // Maximum range of values in the array

void findFrequency(int a[], int n, int freq[]) {
    for(int i = 0; i < MAX; i++) {
        freq[i] = 0;
    }

    for(int i = 0; i < n; i++) {
        freq[a[i]]++;
    }
}
